rosdep/python.yaml

```diff
@@ -3674,6 +3674,17 @@ python-vlc-pip:
   ubuntu:
     pip:
       packages: [python-vlc]
+python-voluptuous:
+  debian:
+    jessie:
```

Comment:
This package is available in [all versions of Debian](https://packages.debian.org/search?keywords=python-voluptuous&searchon=names&exact=1&suite=all&section=all), can you please use the apt version everywhere?